src/main/java/org/schemaspy/service/TableService.java

```diff
@@ -118,8 +116,8 @@ private void initColumnAutoUpdate(Database db, Table table, boolean forceQuotes)
             if (forceQuotes) {
                 if (!table.isLogical()) {
                     // don't completely choke just because we couldn't do this....
-                    LOGGER.warning("Failed to determine auto increment status: " + exc);
-                    LOGGER.warning("SQL: " + sql.toString());
+                    LOGGER.warn("Failed to determine auto increment status: {}", exc);
```

Comment:
I just saw this.
I'm thinking and this is my subjective view.

One should never call logging on the same level twice, for one error.
In our case, there might not be a concurrency issue, but as a rule of thumb.

This is to avoid:
warn - Failed to deter....
Bunch of other loggs from other threads
warn - SQL: ....

This should probably be refactored into:
```
LOGGER.warn("Failed to determine auto increment status using SQL: {}", sql.toString(), exc);
```